fvm/state/accounts.go

```diff
@@ -235,13 +235,17 @@ func contractKey(contractName string) string {
 }
 
 func (a *Accounts) getContract(contractName string, address flow.Address) ([]byte, error) {
+	fmt.Printf("TEMP LOGGING: get contract called for %s.%s \n", address.String(), contractName)
+
 	contract, err := a.ledger.Get(string(address.Bytes()),
 		string(address.Bytes()),
 		contractKey(contractName))
 	if err != nil {
 		return nil, newLedgerGetError(contractName, address, err)
 	}
 
+	fmt.Printf("TEMP LOGGING: a contract returned for %s.%s: %s...%s (len: %d)\n", address.String(), contractName, contract[:30], contract[len(contract)-20:], len(contract))
```

Comment:
what is `contract[:30], contract[len(contract)-20:]`?

Reply:
50 bytes of contract content to check if is accurate